docs/guide/performance.md

```diff
@@ -102,3 +102,20 @@ export default defineConfig({
 Note that you should only warm up files that are frequently used to not overload the Vite dev server on startup. Check the [`server.warmup`](/config/server-options.md#server-warmup) option for more information.
 
 Using [`--open` or `server.open`](/config/server-options.html#server-open) also provides a performance boost, as Vite will automatically warm up the entry point of your app or the provided URL to open.
+
+## Use Lesser or Native Tooling
+
+Keeping Vite fast with a growing codebase is about reducing the amount of work for the source files (JS/TS/CSS).
+
+Examples of doing less work:
+
+- Using CSS instead of SaaS when possible (nesting can be handled by postcss)
+- Don't transform SVG to React components
```

Comment:
Yeah this was I'm using (https://github.com/ArnaudBarre/vite-plugin-fast-react-svg), but I not sure it's solid enough to be reomanded as a default instead of svgr. What I only know is that svgr is too slow